server/scripts/2019TaxSeasonDataScript.php

```diff
@@ -0,0 +1,361 @@
+
+		// Thursdays
+		$dates = array('2019-02-07', '2019-02-14', '2019-02-21', '2019-02-28', '2019-03-07', '2019-03-14', '2019-03-21', '2019-03-28', '2019-04-04', '2019-04-11');
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 16:00:00", "$date 19:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 16:00:00", 200, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 200, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 200, 60, $siteId);
+		}
+
+		// Sundays
+		$dates = array('2019-02-03', '2019-02-10', '2019-02-17', '2019-02-24', '2019-03-03', '2019-03-10', '2019-03-17', '2019-03-24', '2019-03-31', '2019-04-07');
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 13:00:00", "$date 16:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 200, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 200, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 200, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+		insertSiteRoleLimit(6, $preparerRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Center for People in Need data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertBennettMartinLibraryData() {
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Bennett Martin data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = insertSite('Bennett Martin Library', '14th & N Street', '402-472-9638', FALSE, FALSE);
+
+		// Sundays
+		$dates = array('2019-02-03', '2019-02-10', '2019-02-17', '2019-02-24', '2019-03-03', '2019-03-10', '2019-03-17', '2019-03-24');
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 13:00:00", "$date 16:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 200, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 200, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 200, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+		insertSiteRoleLimit(4, $preparerRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Center for People in Need data', MY_EXCEPTION);
+		die();
+	}
+
+}
+
+function insertInternationalStudentScholarSiteData() {
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The ISS data has already been inserted');
+	}
+
+	// TODO: NEED TO FINISH UPDATED STATION LIMITING CODE BEFORE I CAN ENTER THIS
```

Comment:
Actually, I will probably go ahead and put in the data for all the other sites through this script, and then come back to this later--international appointments don't start until March sometime, so we have time to not put in the data now.